### ISSUE TITLE
StorageNotFound error during proof

### ISSUE THREAD

**Issue by nrempel** [hyperledger/aries-cloudagent-python#319]
This error occurs during proof-request proof flow

```
tob-agent_1    | Traceback (most recent call last):
tob-agent_1    |   File "/home/indy/.pyenv/versions/3.6.9/lib/python3.6/site-packages/aries_cloudagent/core/dispatcher.py", line 169, in handle_message
tob-agent_1    |     await handler(context, responder)
tob-agent_1    |   File "/home/indy/.pyenv/versions/3.6.9/lib/python3.6/site-packages/aries_cloudagent/protocols/present_proof/v1_0/handlers/presentation_ack_handler.py", line 36, in handle
tob-agent_1    |     await presentation_manager.receive_presentation_ack()
tob-agent_1    |   File "/home/indy/.pyenv/versions/3.6.9/lib/python3.6/site-packages/aries_cloudagent/protocols/present_proof/v1_0/manager.py", line 456, in receive_presentation_ack
tob-agent_1    |     "connection_id": self.context.connection_record.connection_id,
tob-agent_1    |   File "/home/indy/.pyenv/versions/3.6.9/lib/python3.6/site-packages/aries_cloudagent/messaging/models/base_record.py", line 239, in retrieve_by_tag_filter
tob-agent_1    |     raise StorageNotFoundError("Record not found")
tob-agent_1    | aries_cloudagent.storage.error.StorageNotFoundError: Record not found
tob-agent_1    | 2020-01-03 00:00:36,111 aries_cloudagent.core.dispatcher ERROR Handler error: Dispatcher.handle_message
tob-agent_1    | Traceback (most recent call last):
tob-agent_1    |   File "/home/indy/.pyenv/versions/3.6.9/lib/python3.6/site-packages/aries_cloudagent/core/dispatcher.py", line 169, in handle_message
tob-agent_1    |     await handler(context, responder)
tob-agent_1    |   File "/home/indy/.pyenv/versions/3.6.9/lib/python3.6/site-packages/aries_cloudagent/protocols/present_proof/v1_0/handlers/presentation_ack_handler.py", line 36, in handle
tob-agent_1    |     await presentation_manager.receive_presentation_ack()
tob-agent_1    |   File "/home/indy/.pyenv/versions/3.6.9/lib/python3.6/site-packages/aries_cloudagent/protocols/present_proof/v1_0/manager.py", line 456, in receive_presentation_ack
tob-agent_1    |     "connection_id": self.context.connection_record.connection_id,
tob-agent_1    |   File "/home/indy/.pyenv/versions/3.6.9/lib/python3.6/site-packages/aries_cloudagent/messaging/models/base_record.py", line 239, in retrieve_by_tag_filter
tob-agent_1    |     raise StorageNotFoundError("Record not found")
tob-agent_1    | aries_cloudagent.storage.error.StorageNotFoundError: Record not found
```

**Comment by dbluhm** [comment]
I also noticed this but this should be fixed with #314

**Comment by swcurran** [comment]
Closing this issue - addressed with #314.
